frontend/src/components/DisplayIcon.tsx

```diff
@@ -30,20 +22,9 @@ export default function DisplayIcon({ item, icons }: { item: string; icons: Icon
     .filter(Boolean)
     .join(' ')
 
-
-  const iconClassName = [
-    isSocialMedia ? brandColors[item.toLowerCase()] : 'text-gray-600 dark:text-gray-300',
-    'transition-transform transform hover:scale-110',
-    item === 'stars_count' || item === 'starsCount' ? 'icon-rotate' : '',
```

Comment:
These were set here specifically for the icons under the title of the card for forks, starts, etc. There were a hover effect on those. 
This is how they looked before this change:
https://www.awesomescreenshot.com/video/38208560?key=126524f2beea68a4a8ed59632b875d69

This is how they look now, with changed color and no more transform effect:
https://www.awesomescreenshot.com/video/38208572?key=35ab4940071d819861f21906cf9465e8

I'd prefer to keep it as it was for these icons (for both color and effect)